.travis.yml

```diff
@@ -3,13 +3,11 @@ os:
     - linux
     - osx
 julia:
-    - 0.5
-    - 0.6
+    - 1.0
```

Comment:
also 0.7? (as in REQUIRE)

Reply:
General recommendation was no 0.7 testing if 1.0 passes.